### PR TITLE
Chapter 6 sweep

### PR DESCRIPTION
**Pull request by bast**
- TC is in sync and ready to go once CI agrees.

### CODE REVIEW

File: chapter-06/recipe-02/abstract.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+In this recipe, we generate `print_info.c` from the template `print_info.c.in`
```
Comment (robertodr) [inline chapter-06/recipe-02/abstract.md]: :+1: For starting to write abstracts for recipes!

File: chapter-06/recipe-02/abstract.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+configure time.
+
+We should point out that this recipe has a serious limitation and cannot
+emulate `configure_file` fully. The approach that we present here cannot
+generate an automatic dependency which would regenerate `print_info.c` at build
+time. In other words, if you remove the generated `print_info.c` after the
+configure step, this file will not be regenerated and the build step will fail.
+To proper mimic the behavior of `configure_file` we would
+require `add_custom_command` and `add_custom_target`, which we will use in the
+subsequent recipe 3, Generating source code at build-time using Python, where
```
Comment (robertodr) [inline chapter-06/recipe-02/abstract.md]: Just playing devil's advocate here, but can you style the title of the recipe in italics?

File: chapter-06/recipe-02/abstract.md (hunk 1 above)
Comment (bast) [reply chapter-06/recipe-02/abstract.md]: I will add more soon but here we needed one urgently.

File: chapter-06/recipe-02/abstract.md (hunk 2 above)
Comment (bast) [reply chapter-06/recipe-02/abstract.md]: OK, upcoming.

**Pull request merged**